src/flask/cli.py

```diff
@@ -976,6 +976,12 @@ def routes_command(sort, all_methods):
 
 
 def main():
+    if int(click.__version__[0]) < 8:
+        warnings.warn(
+            "Using the `flask` cli with Click 7 is deprecated and"
+            " will not be supported starting with Flask 2.1.",
```

Comment:
Should add something about upgrading to Click 8.

Reply:
done